core/core.go

```diff
@@ -327,6 +334,28 @@ func NewTextile(ctx context.Context, conf Config, opts ...Option) (*Textile, err
 		}
 	}
 
+	jobFinalizedEvents := make(chan archive.JobEvent)
+	t.archiveTracker, err = tracker.New(
+		t.collections,
+		t.bucks,
+		t.pc,
+		t.internalHubSession,
+		conf.ArchiveJobPollIntervalSlow,
+		conf.ArchiveJobPollIntervalFast,
+		jobFinalizedEvents,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	filRetrievalDS := kt.WrapTxnDatastore(t.ts, ktipfs.PrefixTransform{
+		Prefix: datastore.NewKey("buckets/filretrieval"),
+	})
```

Comment:
@sanderpick, I believe this might already exist in the branch you're working on; so this might need to get changed if you're planning to re-use the go-datastore with other wrap-namespace or similar?

In any case, adjusting this to fit your plans should be fine.